ingredients/LSSTCam-imSim/DRP.yaml

```diff
@@ -110,6 +110,9 @@ subsets:
       - matchCatalogsTract
```

Comment:
Aren't the new tasks meant to replace this?

Reply:
That one stays around as a "default" (with a somewhat conservative S/N>50 cut).

Reply:
Is anything using that? I'm a little hesitant about creating datasets that aren't being used.

Reply:
Yeah, the photometric repeatability metrics (PA1, PF1) use it.